package.json

```diff
@@ -18,8 +18,8 @@
   },
   "license": "BSD-3-Clause",
   "peerDependencies": {
-    "react": "^0.14.0",
-    "react-dom": "^0.14.0"
+    "react": "^15.0.0",
+    "react-dom": "^15.0.0"
```

Comment:
Can you make these ranges to instead? `^0.14.0 || ^15.0.0` should be good. Since there are no code changes requires we're compatible with both versions.